Makefile

```diff
@@ -15,13 +15,21 @@ test-coverage:
 format:
 	poetry run isort apps
 	poetry run black apps
+ifeq ($(backend-only),true)
+	echo "skipping frontend format"
+else
 	npm run format
+endif
 
 lint:
 	poetry run flake8 apps
 	poetry run isort --check-only --diff apps
 	poetry run black --check --diff apps
+ifeq ($(backend-only),true)
+	echo "skipping frontend linting"
+else
 	npm run lint
+endif
```

Comment:
How about splitting these into `format-backend`, `format-frontend`, `lint-backend`, `lint-frontend`, and then making `format` and `lint` call both the `backend` and `frontend` variants?

Kind of like bakerydemo: https://github.com/wagtail/bakerydemo/blob/main/Makefile

Reply:
Yeah we did a similar thing in the Wagtail repo also (split into different make commands), but I recall that being not ideal as it added so many different commands.

Reply:
will do this though